fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt

```diff
@@ -0,0 +1,184 @@
+                    
+                    // this  will not be triggered
+                    LayoutInflater123 li123 = LayoutInflater123.from(requireContext()); 
+                    return super.onCreateView(inflater, container, savedInstanceState);
+                }
+            }
+            """
+    ).indented()
+
+    private val fragmentStubJava = java(
+        """
+            package foo;
+            import android.os.Bundle;
+            import android.view.LayoutInflater;
+            import android.view.View;
+            import android.view.ViewGroup;
+            import androidx.annotation.NonNull;
+            import androidx.annotation.Nullable;
+            import androidx.fragment.app.DialogFragment;
+            
+            public class TestFragment extends Fragment {
+            
+                @Nullable
+                @Override
+                public View onCreateView(@NonNull LayoutInflater inflater, 
+                                            @Nullable ViewGroup container, 
+                                            @Nullable Bundle savedInstanceState) {
+                    LayoutInflater li = LayoutInflater.from(requireContext());
+                    return super.onCreateView(inflater, container, savedInstanceState);
+                }
+            }
+            """
+    ).indented()
+
+    private val dialogFragmentStubKotlin = kotlin(
+        """
+            package foo
+            import android.os.Bundle
+            import android.view.LayoutInflater
+            import android.view.View
+            import android.view.ViewGroup
+            import androidx.annotation.NonNull
+            import androidx.annotation.Nullable
+            import androidx.fragment.app.DialogFragment
+            
+            class TestFragment : DialogFragment() {
+
+                override fun onCreateView(inflater: LayoutInflater, 
+                                            container: ViewGroup?, 
+                                            savedInstanceState: Bundle?): View? {
+                    val li = LayoutInflater.from(requireContext())
+                    return super.onCreateView(inflater, container, savedInstanceState)
+                }
+            }
+            """
+    ).indented()
+
+    private val fragmentStubKotlin = kotlin(
+        """
+            package foo
+            import android.os.Bundle
+            import android.view.LayoutInflater
+            import android.view.View
+            import android.view.ViewGroup
+            import androidx.annotation.NonNull
+            import androidx.annotation.Nullable
+            import androidx.fragment.app.DialogFragment
+            
+            class TestFragment : Fragment() {
+
+                fun someFunction() {
+                    val li = LayoutInflater.from(requireContext())
+                }
+            }
+            """
+    ).indented()
+
+    @Test
+    fun `pass java check`() {
```

Comment:
Lets rename the tests to what we expect.

For this one `java expect fail dialog fragment with fix`